CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 ## Current Develop Branch
 
+## 4.12.0 Tuesday September 25 2018
```

Comment:
We should also update this with today's date

Reply:
Thank you for the eagle eyes!